python/nmodl/__init__.py

```diff
@@ -3,8 +3,10 @@
 
 if sys.version_info >= (3, 9):
     from importlib.resources import files
+    from importlib.metadata import version, PackageNotFoundError
 else:
     from importlib_resources import files
+    from importlib_metadata import version, PackageNotFoundError
```

Comment:
```suggestion
    from importlib_metadata import version
```
would this work?

Reply:
`importlib_metadata` stuff is now irrelevant in the init because nothing uses it anymore, my linter just didn't report it so I forgot to remove it.